docs/self-managed/setup/overview.md

```diff
@@ -31,7 +31,7 @@ For details on supported environments (e.g. Java or Elasticsearch versions), see
 
 You have the following options to run or deploy the above components in a self-managed fashion:
 
-- **Helm/Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. In addition to stock Kubernetes we also officially support variety of providers like Red Hat OpenShift and Amazon EKS. Also using Kubernetes with Minikube or KIND can be an interesting environment to run Camunda 8 locally on developer machines.
+- **Kubernetes**: We strongly recommend using Kubernetes to run Camunda 8 in production. We provide supported HELM charts to deploy Camunda 8 into your clusters. We suport [CNCF Certified Kubernetes disctributions](https://www.cncf.io/training/certification/software-conformance/) including a variety of providers like RedHat OpenShift and Amazon EKS. Consider using Minikube or KIND to run Camunda 8 locally on developer machines.
```

Comment:
why do we need this part at all? wouldn't it make sense to move it into `supported-environments.md` and not to repeat myself? especially with the new headers approach I propose